app.py

```diff
@@ -26,3 +29,14 @@ def data():
 @app.route('/<path:path>')
 def send_js(path):
     return send_from_directory('web/src', path)
+
+
+@app.route('/')
+def index():
+    return redirect(url_for('data'))
+
```

Comment:
@javatarz what should be the response for base URL?

Reply:
the `/` route should probably redirect to  `/index.html` because that renders the web page. Sounds like a good smart default to have :)

Reply:
the `/` route should probably redirect to `/index.html` because that renders the web page. Sounds like a good smart default to have :)